### DISCUSSION TITLE
New Releases?

### DISCUSSION THREAD

**Discussion by RockSolidSW** [dnnsoftware/Dnn.Platform#5138] [General] [answered]
Hi Everyone, will there be new releases of DNN Platform soon?  It's been quite a while.  Thanks!

**Comment by valadas** [comment, marked as answer]
9.11.0 is in the works with the possibility of a 9.10.x if there is some huge critical issue.
DNN Platform being open source, the release date totally depends on the contributions we get. For 9.11.0 our goal is to have a complete automation to remove Telerik upon upgrades and/or clean installs without it (as an option). So that we are ready for forcibly remove it in v10. There have been some fairly recent PRs that need testing and/or some adjustments, but I'd say we are close and I would guess a release is nearing (a few weeks maybe)...

**Reply by mitchelsellers** [reply]
Everything said above!

**Reply by barnsley7** [reply]
does this mean that in version 9.11, users will no longer have to remove Telerik manually as described here: 
https://dnnsupport.dnnsoftware.com/hc/en-us/articles/4408761675666-How-to-manually-remove-Telerik-from-Evoq-DNN-platform

**Reply by bdukes** [reply]
In 9.11, the upgrade wizard will prompt you to automatically remove Telerik, which will follow the steps listed at https://docs.dnncommunity.org/content/getting-started/setup/telerik-removal/index.html (which is more up-to-date and complete than the article you referenced). Upon upgrade to DNN 10.0.0, the plan is that Telerik removal will happen automatically without a prompt.

**Reply by RockSolidSW** [reply]
Awesome, thanks @valadas !

**Reply by uzmannazari** [reply]
Why is version 9.10.3 not released even though it is closed?

**Comment by wbonekamp** [comment]
Can you give us a status update?

Is there a planning for the new release?

**Reply by mitchelsellers** [reply]
We are targeting a RC in the next week, or two.  We have a single pending PR that once it comes in we can cut the RC
